test/TradeTrustERC721.js

```diff
@@ -150,5 +151,23 @@ contract("TradeTrustErc721", accounts => {
         /VM Exception while processing transaction: revert Cannot send token: Token not owned by token registry/
       );
     });
+
+    it("should be able to send token to new title escrow", async () => {
+      const currentTokenOwner = await tokenRegistryInstanceWithShippingLineWallet.ownerOf(merkleRoot);
+      expect(currentTokenOwner).to.deep.equal(tokenRegistryAddress);
+
+      await tokenRegistryInstanceWithShippingLineWallet.sendToNewTitleEscrow(owner1, holder1, merkleRoot, {from: shippingLine})
+      const nextTokenOwner = await tokenRegistryInstanceWithShippingLineWallet.ownerOf(merkleRoot);
+      expect(nextTokenOwner).to.not.deep.equal(currentTokenOwner);
```

Comment:
This checks that is not the same owner, but doesnt really check that it's a title escrow (with right bene & holder)

Reply:
done. added the portion that checks if the new title escrow created and sent back to, is assigned with the correct bene and holder.